File: C#/new_approach/new_approach.cs
using System;

namespace ReverseNumber
{
	class Program
	{
	    public class Reverse
	    {
	        public string ReverseNum(string num)
	        {
	            StringBuilder return_str = new StringBuilder(num);
	            int i = 0;
	            int len = num.Length;
	            int symmetry;
	            char temp;
	            
	            for (i = 0; i < len / 2; i++)
	            {
	                symmetry = len - 1 - i;
	                temp = return_str[i];
	                return_str[i] = return_str[symmetry];
	                return_str[symmetry] = temp;
	            }

	            return return_str.ToString();
	        }
	    }

		static void Main(string[] args)
		{
			Reverse rev = new Reverse();
			string num = Console.ReadLine();
			string res = rev.ReverseNum(num);
			Console.WriteLine(res);
		}
	}
}

File: Java/new_approach/new_approach.java
import java.util.Scanner;

public class Reverse
{
	public String reverseNum(String num)
	{
		char[] number = num.toCharArray();
		StringBuilder return_str = new StringBuilder();
		int len = number.length;
		int bound = len / 2;
		int i = 0;
		int symmetry = len - 1 - i;
		char temp;

		for (i = 0; i < bound; i++)
		{
			symmetry = len - 1 - i;
			temp = number[i];
			number[i] = number[symmetry];
			number[symmetry] = temp;
		}

		return_str.append(number);
		return return_str.toString();
	}
}

class ReverseNum
{
	public static void main(String[] args)
	{
		Scanner in = new Scanner(System.in);
		Reverse rev = new Reverse();
		String num = in.next();
		String res = rev.reverseNum(num);
		System.out.println(res);
	}
}